python-avd/pyavd/api/ip_addressing/__init__.py

```diff
@@ -331,3 +331,24 @@ def evpn_underlay_l2_multicast_group(
         """Return IP address to be used for EVPN underlay L2 multicast group."""
         offset = vlan_id - 1 + underlay_l2_multicast_group_ipv4_pool_offset
         return get_ip_from_pool(underlay_l2_multicast_group_ipv4_pool, 32, offset, 0)
+
+    def get_wan_ha_ip_address(self, local: bool) -> str | None:
```

Comment:
We do not call these methods `get_xxx`. Also I would prefer to have two methods similar to uplinks, mlag etc. (I am aware that they will be very similar, but it will be slightly more efficient).

```suggestion
    def wan_ha_ip_address(self, local: bool) -> str | None:
```

Reply:
this is a copy of an exitsting method. I can split it in two method though

Reply:
also there is no primary or secondary so thats a bit tricky

Reply:
fixed